src/amo/components/SearchResult/styles.scss

```diff
@@ -195,7 +195,7 @@ $icon-default-size: 32px;
 }
 
 .SearchResult-users-icon {
-  height: 10px;
+  height: 12px;
```

Comment:
would it be better expressed as a unit linked to the font size though?  e.g. `1em`? @bobsilverberg

Reply:
It's an interesting idea, but we don't use `em` anywhere else on the site, so I think we need to think about it before introducing something like that.